collectors/ebpf.plugin/README.md

```diff
@@ -191,6 +191,61 @@ The eBPF collector enables and runs the following eBPF programs by default:
 -   `network viewer`: This eBPF program creates charts with information about `TCP` and `UDP` functions, including the
     bandwidth consumed by each.
 
+### `[network viewer]`
+
+You can configure the information shown on `outbound` and `inbound` charts with the settings in this section. 
+
+```conf
+[network viewer]
+    maximum dimensions = 500
+    ports = 1-1024 !145 !domain
+    hostnames = !example.com
+    ips = !127.0.0.1/8 10.0.0.0/8 172.16.0.0/12 192.168.0.0/16 fc00::/7
+```
+
+When you define a `ports` setting, Netdata will collect network metrics for that specific port. For example, if you
+write `ports = 19999`, Netdata will collect only connections for itself. The `ports`, `hostnames`, and `ips` settings
+all accept [simple patterns](/libnetdata/simple_pattern/README.md).
```

Comment:
> The `ports`, `hostnames`, and `ips` settings
all accept [simple patterns]

Simple patterns logic: **return on the first match** (positive or negative match).

If i understand your example correctly it doesnt work like simple patterns.

> ports = 1-1024 !145 !domain

if we return on the first match we dont exclude `145`.

Reply:
So, is order a thing for ips, ports, etc?

Reply:
It is complicated to do 100% like `simple patterns` here, because users could define `ports = 1-1024 3128-5432 40000-65535`, so we have to test for all these acceptable ranges and we would exclude in the others. The same thing happens with IPS. 

About the order, we test the correct order when the values are given in a range, but it does not matter if users define something like this `192.168.0.0/16 172.20.0.0/12`.

Reply:
i dont get it.

`10.0.0.0/8 !10.0.0.0-10.0.0.10`

Does 10.0.0.3 match?

Reply:
> 10.0.0.0/8 !10.0.0.0-10.0.0.10

Does 10.0.0.3 match?

For this example `10.0.0.3` will be eliminated together all values in the range `10.0.0.0-10.0.0.10`, but we continue accepting all the remaining values.

Reply:
ok, so it is

> match at least one positive "pattern" AND do not match any of negated "patterns"

the order is not relevant